examples/api/test/material/scrollbar/scrollbar.0_test.dart

```diff
@@ -18,4 +18,24 @@ void main() {
 
     expect(tester.takeException(), isNull);
   }, variant: TargetPlatformVariant.all());
+
+  testWidgets('The scrollbar should be painted when the user scrolls', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.ScrollbarExampleApp(),
+    );
+    await tester.pump();
+    await tester.pump(const Duration(milliseconds: 10)); // Wait for the thumb to start appearing.
+
+    final Finder scrollbarFinder = find.byType(Scrollbar).last;
+
+    expect(find.text('item 0'), findsOne);
+    expect(find.text('item 9'), findsNothing);
+    expect(scrollbarFinder, isNot(paints..rect()));
+
+    await tester.fling(find.byType(Scrollbar).last, const Offset(0, -300), 10.0);
+
+    expect(find.text('item 0'), findsNothing);
+    expect(find.text('item 9'), findsOne);
+    expect(scrollbarFinder.last, paints..rect());
```

Comment:
Here you're using `scrollbarFinder` but not in the second test. 

We should be consistent, i would suggest to not use `scrollbarFinder` as `find.byType(Scrollbar).last` is already simple enough.

Reply:
Sure, I deleted it in [test: Remove unnecessary scrollbarFinder](https://github.com/flutter/flutter/pull/151463/commits/811773e0f922c5057bfbb19fca5d388b044f05bf)